HerPublicWebsite.BusinessLogic/Services/QuestionFlow/QuestionFlowService.cs

```diff
@@ -308,6 +323,10 @@ private QuestionFlowStep ConfirmLocalAuthorityForwardDestination(Questionnaire q
             {
                 return QuestionFlowStep.SelectLocalAuthority;
             }
+            else if (questionnaire.LocalAuthorityHug2Status is LocalAuthorityData.Hug2Status.NotParticipating)
+            {
+                return QuestionFlowStep.NotParticipating;
+            }
```

Comment:
It doesn't make a difference in practice, but you've used a different order of checking the status for `SelectAddressForwardDestination()` and `ConfirmLocalAuthorityForwardDestination()`.

Please can you commonise so that they are the same order? Given my comment above about the order of the statuses, let's have the `NotTakingPart` route first, so matching the `SelectAddressForwardDestination()`

While it doesn't make a practical difference, it might be enough to briefly trip up a future developer trying to figure our if there was a signigficance to the difference.